### PR TITLE
story/GAT-1133:SEARCH: Add tooltip to the Alliance shield

### PR DESCRIPTION
**Pull request by kandaj**

### CODE REVIEW

File: src/pages/commonComponents/relatedObject/Dataset/Dataset.jsx [hunk 1]
```diff
@@ -35,16 +37,19 @@ const Dataset = ({
     const getPublisherDetails = useCallback(() => {
         const publisher = { name: '', label: '', showShield: false };
         if (!isEmpty(data.datasetv2)) {
+            console.log(data);
```
Comment (kymmeh) [inline src/pages/commonComponents/relatedObject/Dataset/Dataset.jsx:40]: Console.log left in code ^^

File: src/pages/commonComponents/relatedObject/Dataset/Dataset.jsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 /** @jsx jsx */
 import { jsx } from '@emotion/react';
+import Button from 'react-bootstrap/Button';
```
Comment (kymmeh) [inline src/pages/commonComponents/relatedObject/Dataset/Dataset.jsx:3]: Is this being used?

File: src/pages/commonComponents/relatedObject/Dataset/Dataset.jsx [hunk 3]
```diff
@@ -112,11 +113,17 @@ const Dataset = ({
...
-                        data-testid={`publisher-${publisherDetails.name}`}
-                    >
+                        data-testid={`publisher-${publisherDetails.name}`}>
                         {' '}
                         {publisherDetails.name}{' '}
                     </span>
+                    {publisherDetails.showShield && (
+                        <ToolTip text={`Member of ${publisherDetails.memberOf}`}>
+                            <span>
+                                <SVGIcon name='shield' fill='#475da7' className='svg-16 mr-2' viewBox='0 0 16 16' />
```
Comment (kymmeh) [inline src/pages/commonComponents/relatedObject/Dataset/Dataset.jsx:123]: I'm not sure what the approach is to icons, but below the LockSVG has been imported. Should the same apply for this?

Comment (kymmeh) [reply src/pages/commonComponents/relatedObject/Dataset/Dataset.jsx:123]: It seems there's an Icon component and a directory `/images/icons` where new svgs are being added:

`<Icon svg={<IconName />} />`

Comment (kandaj) [reply src/pages/commonComponents/relatedObject/Dataset/Dataset.jsx]: fixed

Comment (kymmeh) [reply src/pages/commonComponents/relatedObject/Dataset/Dataset.jsx]: The icon now appears to be a bit misaligned:

Screenshot 2022-08-09 at 13 16 47

Comment (kymmeh) [reply src/pages/commonComponents/relatedObject/Dataset/Dataset.jsx]: Resolved:

Screenshot 2022-08-10 at 10 40 26

File: src/components/ToolTip/ToolTip.jsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+function ToolTip({ text, placement, ...outerProps }) {
+    return (
+        <OverlayTrigger placement={placement} overlay={renderTooltip(text)}>
+            {outerProps.children}
+        </OverlayTrigger>
+    );
+}
+
+ToolTip.propTypes = {
+    text: PropTypes.string,
```
Comment (ThinkingMonkeyLtd) [inline src/components/ToolTip/ToolTip.jsx:17]: Could text be a ReactNode (i.e. component)

**Review by ThinkingMonkeyLtd — APPROVED** [review]

**Review by kymmeh — APPROVED** [review]

**Pull request merged**